nidata/multimodal/hcp/example1.py

```diff
@@ -1,14 +1,10 @@
 import os
-if os.environ.get('NIDATA_HCP_USERNAME') is None:
-    raise Exception("Must define NIDATA_USERNAME and NIDATA_HCP_PASSWD "
-                    "environment variables, or pass username & passwd "
-                    " to HcpDataset() below.")
 
 from nidata.multimodal import HcpDataset
 # 'aws' == amazon web services also OK
-dataset = HcpDataset(fetcher_type='http')
+dataset = HcpDataset(fetcher_type='aws', profile_name='hcp')
 
 # only fetch anatomical files.
-files = dataset.fetch(n_subjects=1, data_types=['anat'])
+files = dataset.fetch(n_subjects=1, data_types=['diff'])
```

Comment:
Rather than change the anatomical example, I'd like to see you create a new diffusion example.